tests/members/test_members_resource.py

```diff
@@ -216,30 +247,36 @@ def test_search(
     assert hit['permissions']['can_update_visible'] is True
     assert hit['visible'] is True
 
-    hit = data['hits']['hits'][1]
+    hit = data['hits']['hits'][1]  # should be last, test sorting
```

Comment:
To test the `name` sorting I added a user that would be in the middle of the two previous ones (S < N, and then the third one has no name and gets "Untilted" when serialized).

However, this gives me the feeling of being too ad-hoc. On the other hand, I'm not sure if it's worth making a full test just for sorting.